Tribler/Main/vwxGUI/GuiUtility.py

```diff
@@ -112,6 +131,7 @@ def ShowPlayer(self):
 
     @forceWxThread
     def ShowPage(self, page, *args):
+        assert wx.Thread_IsMain()
```

Comment:
Who would remove a decorator from a function that is clearly doing GUI stuff? 
Just remove the asserts you added please.